pyproject.toml

```diff
@@ -227,13 +227,16 @@ warn_unused_ignores = true
 
 [tool.numpydoc_validation]
 checks = [
+  "all",
+  # all but the following:
   "ES01", # Not all docstrings need an extend summary.
   "EX01", # Examples: Will eventually enforce
   "GL01", # Contradicts numpydoc examples
   "GL02", # Permit a blank line after the end of our docstring
   "GL03", # Considering enforcing
   "GL06", # Found unknown section
   "GL07", # "Sections are in the wrong order. Correct order is: {correct_sections}",
+  "GL08", # The object does not have a docstring
```

Comment:
All changes in this PR _except_ for the ones here in `pyproject.toml` are automatic. So, if desired, we can create a separate PR with only the rule change, and merge that first. That way, all changes here will be fully automatic, and we can then add this PR (with auto-changes only) to the git blame ignore.